### ISSUE TITLE
NTR: Histidase activity low in skin stratum corneum (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9254]
New term request Histidase activity low in skin stratum corneum (elthat_111113033243):   UMLS: 1856104

**Comment by pnrobinson** [comment]
@MickeySegal 
If this is referring to histidinemia, it is a very rare disorder that apparently is benign and I have seen one reference for this assay from decades ago in PubMed. I am wondering if it is worth adding an HPO term ? is there a current use for this assay?

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000081
- label: Reduced alpha-aminoadipic semialdehyde synthase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: 
- def: Acitivity of alpha-aminoadipic semialdehyde synthase below the lower limit of normal in cultured fibroblasts.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9255
- synonyms: Alpha-aminoadipic semialdehyde synthase low in fibroblasts(NONE)

**Comment by MickeySegal** [comment]
The term created here was for https://omim.org/entry/238700, not for the requested histidinemia: https://omim.org/entry/235800

But I agree that this is now considered a benign condition for which no one would do a skin biopsy, so I am deleting our finding.
